README.md

```diff
@@ -1 +1,8 @@
-# project-library
\ No newline at end of file
+#Project: Martin and Mai's Recipe book
+This project involves building a responsibe web page that displays elements from a provied dataset (recipes) upon loading. Users can interact through features such as filtering, sorting, select recipes from random, search and so on. 
+
+#How we solved the problem
+Divided and conquered. We worked on different functions and then merged them. We established our plan using a Figjam sketch. 
```

Comment:
I wish I could get som more information about how you solved the problem, but I also like how you keep it sweet and simple!